support-frontend/assets/helpers/urls/url.ts

```diff
@@ -12,15 +12,12 @@ const DOMAINS: Record<Env, Domain> = {
 };
 
 // ----- Functions ----- //
-const getQueryParameter = (
-	paramName: string,
-	defaultValue?: string,
-): string | null | undefined => {
-	const params = new URL(window.location).searchParams;
+const getQueryParameter = (paramName: string, defaultValue = ''): string => {
```

Comment:
for future reference, what we didn't spot is that the old defaultValue was `undefined` whereas now it's `''`,  this caused a subtle promo link regression that is now fixed https://github.com/guardian/support-frontend/pull/3402